### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

**Review by gunan — APPROVED** [review]
@asimshankar Does this PR cover all the docs referencing Ubuntu 14?

File: tensorflow/docs_src/install/install_sources.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ following:
 *   [Bazel on Windows](https://bazel.build/versions/master/docs/windows.html)
 *   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/r0.12/tensorflow/contrib/cmake)
 
+Note: Starting from 1.6 release, our prebuilt binaries will use AVX
+instructions. This may break TF on older CPUs.
```
Comment (asimshankar) [inline tensorflow/docs_src/install/install_sources.md]: Perhaps say, "Older CPUs may not be able to execute these binaries"
(and avoid the acronym "TF" on this page)?

File: tensorflow/docs_src/install/install_sources.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ following:
 *   [Bazel on Windows](https://bazel.build/versions/master/docs/windows.html)
 *   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/r0.12/tensorflow/contrib/cmake)
```
Comment (asimshankar) [inline tensorflow/docs_src/install/install_sources.md]: Should this link refer to master instead of release 0.12?

**Review by asimshankar — COMMENTED** [review]
I'm not entirely sure, but I did also notice: tensorflow/contrib/cmake/README.md might need to be rebuilt?

**Review by MarkDaoust — APPROVED** [review]

**Comment by av8ramit** [comment]
@asimshankar is there an automated way to rebuild that file? I'm not sure. I've fixed the other two suggestions.

**Comment by asimshankar** [comment]
Sorry, I don't know why I said "rebuilt" :) I meant updated.

File: tensorflow/docs_src/install/install_sources.md [hunk 3]
```diff
@@ -25,8 +25,10 @@ like to try to build TensorFlow on Windows anyway, use either of the
 following:
 
 *   [Bazel on Windows](https://bazel.build/versions/master/docs/windows.html)
-*   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/r0.12/tensorflow/contrib/cmake)
+*   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/cmake)
```
Comment (asimshankar) [inline tensorflow/docs_src/install/install_sources.md]: Actually, should this point to `master` or the current release version? (Though I guess it doesn't matter a whole lot)

Comment (MarkDaoust) [reply tensorflow/docs_src/install/install_sources.md]: (I'm pretty sure) this is handled automatically by the site-generation tools.

**Pull request merged**